### PR TITLE
iphone-sms-backup-and-restore: Add version 2.14.37

### PR DESCRIPTION
**Pull request by digrouz**

### CODE REVIEW

**Comment by digrouz** [comment]
Hello, How to do this ?

**Comment by digrouz** [comment]
using 'notes' https://github.com/lukesampson/scoop/wiki/App-Manifests ?

**Comment by Ash258** [comment]
You have two options:

1. Add iTunes manifest and suggest it.
1. Add notes with link to it and with explanation why it is needed.

**Comment by digrouz** [comment]
I first added a note.

Regarding iTunes, I guess I have to make another PR ?

File: bucket/iphone-sms-backup-and-restore.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            "url": "https://www.backuptrans.com/download/exe/iphone-sms-backup-and-restore_x64.exe#/dl.7z",
+            "hash": "431555dd10f7c36c771be261051afac65c039c8b8f271fddfa67151dbdcc6747"
+        },
+        "32bit": {
+            "url": "https://www.backuptrans.com/download/exe/iphone-sms-backup-and-restore.exe#/dl.7z",
+            "hash": "5259fa6d271edc5856a1fd43994ae576552b26cd800d80eaa950190124297b45"
+        }
+    },
+    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst*nsis\" -Recurse",
+    "notes": "Apple iTunes is required to use this software. iTunes provides the 'Apple Mobile Device' framework to allow this app to detect iPhones.",
```
Comment (Ash258) [inline bucket/iphone-sms-backup-and-restore.json]:
```suggestion
    "notes": "Without iTunes application (https://support.apple.com/downloads/itunes) installed there is not possible to detect iPhone devices.",
```

**Pull request merged**